__tests__/db.test.js

```diff
@@ -128,3 +128,41 @@ describe("/api/reviews", () => {
       });
   });
 });
+
+describe("/api/reviews/review:id/comments", () => {
+  test("GET - status: 200 - should return correct array of comments objects", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((result) => {
+        expect(result.body.comments.length).toBeGreaterThan(0);
+        result.body.comments.forEach((comment) => {
+          expect(Object.keys(comment).length).toBe(6);
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.review_id).toBe("number");
```

Comment:
Can we check the review_id when using parametric endpoints for ids.

Can we also add a test for a 200 response but where there are no comments for that review_id